src/theme.jl

```diff
@@ -58,88 +58,127 @@ function get_stroke_vector(linestyle::Symbol)
   error("unsupported linestyle: ", linestyle)
 end
 
+using DocStringExtensions
+
+"""
+$(FIELDS)
+"""
 @varset Theme begin
     # If the color aesthetic is not mapped to anything, this is the color that
     # is used.
```

Comment:
I feel like the comments are unnecessary with the docstring update and we should remove them.